src/diffusers/modeling_flax_utils.py

```diff
@@ -280,6 +284,7 @@ def from_pretrained(
         revision = kwargs.pop("revision", None)
         from_auto_class = kwargs.pop("_from_auto", False)
         subfolder = kwargs.pop("subfolder", None)
+        prng_key = kwargs.pop("prng_key", None)
```

Comment:
@patil-suraj - this isn't really needed no? We're only inferring the shape here - so maybe let's also use `jax.random.PRNGKey(0)` no?

Reply:
you are right! handled in https://github.com/huggingface/diffusers/pull/513/commits/803da8f666b2f17c6accf4a73c804fdefd459461